_includes/docs/pe/user-guide/integrations/http.md

```diff
@@ -85,8 +86,9 @@ To do this, check the Enable security checkbox and click on the Headers filter.
 Once the Headers filter has been configured, it will also need to be specified in the uplink message as follows.
 
 ```ruby
--H "test-header:secret"
+-H "test-header:security"
```

Comment:
This should be secret. not security. We use "secret" for an example of the password.